templates/nsfwlanding.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% block title %}NSFW content gated{% endblock %}
+{% block sortstyle %}{% endblock %}
+{% block content %}
+<div id="nsfw_landing">
+    <h1>😱 {% if res_type == crate::utils::ResourceType::Subreddit %}r/{{ res }} is a NSFW community!{% else if res_type == crate::utils::ResourceType::User %}u/{{ res }}'s content is NSFW!{% else if res_type == crate::utils::ResourceType::Post %}This post is NSFW!{% endif %}
+    </h1>
+    <br />
+    <p>Enable "Show NSFW posts" in <a href="/settings">settings</a> to view this {% if res_type == crate::utils::ResourceType::Subreddit %}subreddit{% else if res_type == crate::utils::ResourceType::User %}user's posts or comments{% else if res_type == crate::utils::ResourceType::Post %}post{% endif %}.</p>
```

Comment:
These feel a little verbose all in one line, would splitting them into three conditional h1 tags be better? Or is it common to have lines like this?